lang/julia/README.md

```diff
@@ -0,0 +1,371 @@
+# Julia
+
+In this example, we will cover:
+ - [Pulling a julia container from the library.](#then-pull-the-container-from-the-library)
+ - [Building the container from a recipe.](#to-build-from-a-recipe)
+ - [Running a julia script.](#to-run-a-julia-script)
+
+<br>
+
+Now of corse you can build the [Official julia container from docker](https://hub.docker.com/_/julia/), <br>
+But for this example we will install julia in a [Ubuntu container](https://hub.docker.com/_/ubuntu/). <br>
+
+<br>
+
+#### What you need:
+ - Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim` or `nano`.
+ - Root access (only if you're [building from a recipe](#to-build-from-a-recipe)).
+ 
+
+<br>
+
+____
+
+
+<br>
+
+
+#### To start, make the working directory:
+
+```
+$ mkdir julia
+$ cd julia/
+```
+
+<br>
+
+
+#### Then, pull the container from the library:
+
+```
+$ singularity pull library://sylabs/examples/julia.sif:latest
+```
+
+<br>
+
+#### Rename the container you pulled:
+
+```
+$ mv julia.sif_latest.sif julia.sif
+```
+
+<br>
+
+#### To run the container:
+
+```
+$ singularity run julia.sif 
+Hello world from: https://raw.githubusercontent.com/sylabs/examples/master/lang/julia/hello-world.jl
+For full tutorial, visit: https://github.com/sylabs/examples/lang/julia
+$
+```
+
+<br>
+<br>
+
+### To build from a recipe:
+
+You will need root access to build from a recipe.
+
+#### First, make the definition file: (aka, the recipe)
+
+```
+$ nano julia.def
+```
+```
+BootStrap: docker
```

Comment:
Let's change this to `library`. I'm sure we have an Ubuntu 16 image there.

Reply:
Done. 
Q. is the [sylabs//library/library/default](https://cloud.sylabs.io/library/library/default) like the official release?